test/interactions/pointerInteractionTests.ts

```diff
@@ -1,230 +1,220 @@
+
+        TestMethods.triggerFakeTouchEvent("touchstart", eventTarget, [{x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2}]);
+        TestMethods.triggerFakeTouchEvent("touchstart", eventTarget, [{x: 2 * SVG_WIDTH, y: 2 * SVG_HEIGHT}]);
+        assert.isTrue(callbackCalled, "callback called on exiting Component (touch)");
+        assert.deepEqual(lastPoint, { x: 2 * SVG_WIDTH, y: 2 * SVG_HEIGHT }, "was passed correct point (touch)");
+
+        callbackCalled = false;
+        TestMethods.triggerFakeTouchEvent("touchstart", eventTarget, [{x: 3 * SVG_WIDTH, y: 3 * SVG_HEIGHT}]);
+        assert.isFalse(callbackCalled, "callback not called again if already outside of Component (touch)");
+
+        pointerInteraction.offPointerExit(callback);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, SVG_WIDTH / 2, SVG_HEIGHT / 2);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, 2 * SVG_WIDTH, 2 * SVG_HEIGHT);
+        assert.isFalse(callbackCalled, "callback removed by passing null");
+
+        svg.remove();
+      });
+
+      it("can register two callbacks for the same pointer event", () => {
+        let enterCallback1Called = false;
+        let enterCallback2Called = false;
+
+        let enterCallback1 = () => enterCallback1Called = true;
+        let enterCallback2 = () => enterCallback2Called = true;
+
+        pointerInteraction.onPointerEnter(enterCallback1);
+        pointerInteraction.onPointerEnter(enterCallback2);
+
+        let insidePoint = { x: SVG_WIDTH / 2, y: SVG_HEIGHT / 2 };
+        let outsidePoint = { x: SVG_WIDTH * 2, y: SVG_HEIGHT * 2 };
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, outsidePoint.x, outsidePoint.y);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, insidePoint.x, insidePoint.y);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, outsidePoint.x, outsidePoint.y);
+
+        assert.isTrue(enterCallback1Called, "callback 1 was called on entering Component (mouse)");
+        assert.isTrue(enterCallback2Called, "callback 2 was called on entering Component (mouse)");
+
+        enterCallback1Called = false;
+        enterCallback2Called = false;
+        pointerInteraction.offPointerEnter(enterCallback1);
+
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, outsidePoint.x, outsidePoint.y);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, insidePoint.x, insidePoint.y);
+        TestMethods.triggerFakeMouseEvent("mousemove", eventTarget, outsidePoint.x, outsidePoint.y);
+
+        assert.isFalse(enterCallback1Called, "callback 1 was disconnected from pointer enter interaction");
+        assert.isTrue(enterCallback2Called, "callback 2 is still connected to the pointer enter interaction");
+
+        svg.remove();
+      });
+
+      it("calls all the drag pointer interaction callbacks when needed", () => {
```

Comment:
It's being tested that the interactions behave well when interacting to each other and that one is not overriding the others. However I am not sure about the usefulness of this and it also takes a significant amount of time. Should I delete it? 

The inspiration for it comes from other tests (DragLayerTests, Drag Interaction)